sci-libs/libspatialindex/libspatialindex-1.8.5.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2014 Julian Ospald <hasufell@posteo.de>
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit autotools eutils
+
+MY_PN="spatialindex-src"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="General framework for developing spatial indices"
+HOMEPAGE="http://libspatialindex.github.com/"
+SRC_URI="http://download.osgeo.org/libspatialindex/${MY_P}.tar.bz2"
+LICENSE="MIT"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0/4"
+IUSE="debug static-libs"
+
+S=${WORKDIR}/${MY_P}
+
+src_prepare() {
+	epatch "${FILESDIR}"/${PN}-1.8.1-QA.patch
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable static-libs static) \
+		$(use_enable debug)
+}
+
+src_install() {
```

Comment:
> Btw, is there any reason to not add this ebuild to gx86 directly?

I could ask that question for this whole overlay, no? gx86 workflow sucks and is broken and there is no review process. I'm only adding things there if users complain or if it is overly useful in some way.